docs/src/_guide/integration.md

```diff
@@ -37,12 +37,12 @@ On a shell, you can explicitly `export` environment variables, that will be pass
 ```bash
 export TAP_FOO_BAR=bar
```

Comment:
Perhaps this is best for another issue / PR, but we probably shouldn't recommend folks manage env vars this way. They can use the `env:` key to export environment variables!